src/Parser/MarksmanshipHunter/Modules/Items/Tier19_2p.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SpellUsable from 'Parser/Core/Modules/SpellUsable';
+import RESOURCE_TYPES from 'common/RESOURCE_TYPES';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatNumber } from "../../../../common/format";
```

Comment:
This could be simple `'common/format'`.

Reply:
Not 'could be', but 'should be'. I'm pretty sure you only do relative path for files inside spec, and use absolute path for common / utils

Reply:
Yeah, I think it happens when you allow IDE (at least WebStorm does it) to automatically (or more precisely, from the context menu made by Alt + Enter) import missing things.